specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2024-10-01-preview/workspaceRP.json

```diff
@@ -6808,6 +6978,14 @@
       "description": "The properties of a machine learning workspace.",
       "type": "object",
       "properties": {
+        "agentsEndpointUri": {
+          "description": "The URI of agents endpoint associated with this workspace.",
+          "type": "string",
```

Comment:
>type": "string", [](http://example.com/codeflow?start=11&length=16)

sounds like you can also add a url format to it

Reply:
arm-id should be added to WorkspacePrivateEndpointResource.id, here is the agentsEndpointUri format similar to WorkspacePrivateEndpointResource.id ? If yes you should change the prop name to something like agentsEndpointId, otherwise use url as format

Reply:
fixed as url format